### PR TITLE
support refinitiv styles

### PR DESCRIPTION
**Pull request by kevindavi5**

### CODE REVIEW

**Comment by kevindavi5** [comment]
@jpeters5392, do you think it could be pulling the prettier config from somewhere else? Seems like a lot of changes.

**Comment by kevindavi5** [comment]
> @jpeters5392, do you think it could be pulling the prettier config from somewhere else? Seems like a lot of changes.

It must have been using the prettier defaults. I added the prettier config from docz to the project and that fixed it.

File: src/components/shared/Search/SearchContext.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    this.state = {
+      menus: [],
+      searching: false,
+      setMenus: this.setMenus,
+      setSearching: this.setSearching,
+    }
+  }
+
+  public componentDidMount(): void {
+    const initial = getMenusFromDocs(this.props.docs)
```
Comment (jpeters5392) [inline src/components/shared/Search/SearchContext.tsx]: Why set these here and not the constructor?

**Review by jpeters5392 — APPROVED** [review]

File: src/components/shared/Search/SearchContext.tsx (hunk 1 above)
Comment (kevindavi5) [reply src/components/shared/Search/SearchContext.tsx]: Fixed.

**Pull request merged**